### PR TITLE
Add Zoho CRM REST connector

### PR DESCRIPTION
**Pull request by Chuhaa**
# Description
Add Zoho CRM REST connector

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

**Test Configuration**:
* Ballerina Version: 2201.0.3
* Operating System: Ubuntu
* Java SDK: 11

# Checklist:

### Changes to OpenAPI definition

- [x] Modified `info` section of the OpenAPI definition file - [Example](https://github.com/ballerina-platform/ballerina-extended-library/discussions/74)

    - `description` - what the connector is about 
    - `x-ballerina-init-description` OpenAPI extension - connector initialization details 
    - `x-ballerina-display` OpenAPI extension - connector name and path to the connector icon 

### Changes to connector module

- [x] Added license header at the top of each .bal file. 
- [x] Added Package.md as per the guide [here](https://github.com/ballerina-platform/ballerina-extended-library/discussions/77)
- [x] Added Module.md for the module of the  connector as per the guide [here](https://github.com/ballerina-platform/ballerina-extended-library/discussions/78)
- [x] Added connector icon to the root of the connector. Icon needs to be a `png` of 200x200 px size with name `icon.png`
- [x] Added Ballerina.toml file with following information. For keywords refer to [guide](https://github.com/ballerina-platform/ballerina-extended-library/discussions/72)

### Verifying connector module

- [x] Compiled the connector successfully with the targeted ballerina version. 
- [x] Conducted smoke tests on the connector (Recommended when OpenAPI definition is obtain from an unofficial source)

### Security checks
 - [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: openapi/zoho.crm.rest/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["Sales & CRM/Customer Relationship Management", "Cost/Paid"]
+org = "ballerinax"
+name = "zoho.crm.rest"
+icon = "icon.png"
+distribution = "2201.0.3"
+repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
+version = "1.0.1"
```
Comment (RolandHewage) [inline openapi/zoho.crm.rest/Ballerina.toml:9]: Shall we go with 1.0.0. Rebuild after doing the change.

Comment (Chuhaa) [reply openapi/zoho.crm.rest/Ballerina.toml]: Fixed in https://github.com/ballerina-platform/openapi-connectors/commit/ff0eb5a6dd70aefdc38bb071a3152c9930f30b9f

**Review by LakshanSS — COMMENTED** [review]
There is a typo in the name of the openapi yaml file.
Shall we update it?

**Comment by Chuhaa** [comment]
> There is a typo in the name of the openapi yaml file. Shall we update it?

Fixed in  https://github.com/ballerina-platform/openapi-connectors/commit/26cc842d42343000661b3c4db23f74de4702532c

File: openapi/zoho.crm.rest/Module.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+### Step 3: Invoke connector operation
+1. Now you can use the operations available within the connector. Note that they are in the form of remote operations.
+
+    Following is an example to add a record in Zoho. 
+
+    ```ballerina
+    public function main() returns error? {
+        Response response = check zohoClient->addRecord("contacts", {data:[{Last_Name: "NC"}]});
```
Comment (LakshanSS) [inline openapi/zoho.crm.rest/Module.md:42]: Shouldn't this be `rest:Response`?
```suggestion
        rest:Response response = check zohoClient->addRecord("contacts", {data:[{Last_Name: "NC"}]});
```

Comment (Chuhaa) [reply openapi/zoho.crm.rest/Module.md:42]: Fixed in https://github.com/ballerina-platform/openapi-connectors/commit/2070967a7e542a3c68c479ad4bff648d3f8d8b64

**Review by LakshanSS — APPROVED** [review]

**Review by SkNuwanTissera — APPROVED** [review]

**Pull request merged**